.travis.yml

```diff
@@ -37,11 +41,14 @@ before_script:
   - make install
   - popd
   - popd
-  - mkdir build
-  - cd build
+  - pip install --user -r requirements.txt
```

Comment:
You should be able to install without --user, I think even without sudo

Reply:
I fought *a lot* with Travis; at one stage I certainly needed the `--user`- but maybe it can be removed now- will try.

Reply:
You don't need sudo in if the language is set to python. But since this is set to c++ (?), you might need --user, or sudo.